Tony-Ndichu/StackOverflow-Lite
# [Chore #159678382] Consistent design between sign-in and sign-up pages

**What does this PR do?**

- This PR ensures that the design between the login and sign-up page is consistent.

**How to test it**
```
$ git clone url
$ git checkout chore-ensure-consistent-design-auth-pages-#159678382

```

- Navigate UI folder into views folder

-Compare Sign-Up.html with Sign-In.html

**Relevant pivot tracker story**
[#159678382](https://www.pivotaltracker.com/story/show/159678382)

**Relevant screenshots **
![screenshot 99](https://user-images.githubusercontent.com/37045046/43934882-09ddc77a-9c5a-11e8-94ff-ecae9bc0375b.png)
![screenshot 100](https://user-images.githubusercontent.com/37045046/43934884-0a209316-9c5a-11e8-855f-aff5d439a5ae.png)
